src/components/signup/index.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+import React from 'react';
+import Grid from 'material-ui/Grid';
+import Typography from 'material-ui/Typography';
+import Paper from "material-ui/Paper";
+import Button from 'material-ui/Button';
+import TextField from "material-ui/TextField";
+import "./signup.css";
+
+const SignUpForm = (props: Object) : HTMLDivElement => (
+  <div className="login-grid__container">
+     <Grid container spacing={24}>
+      <Paper className="paper">
+        <Typography className="heading" type="headline" component="h1">
+          SIGN UP
+        </Typography>
+        <form onSubmit={(event) => { props.handleSubmit(event) }}>
```

Comment:
`props.handleSubmit` can be directly passed to the `onSubmit`, like
```javascript
<form onSubmit={props.handleSubmit}>
```

Reply:
I need to capture the event in order to prevent the default action.